utils/phraseGenerator.go

```diff
@@ -0,0 +1,636 @@
+package utils
```

Comment:
This is the Docker list, isn't it? Is there a reason why we can't import their package, but have to copy it over? Is that fine with Docker's (and our) IP Licensing?

Reply:
I wanted our own list we could customize.  No headers were stripped from the original.